WinUIGallery/DataModel/ControlInfoData.json

```diff
@@ -2009,6 +2009,24 @@
             }
           ]
         },
+        {
+          "UniqueId": "CaptureElementPreview",
+          "Title": "Capture element / camera preview",
```

Comment:
Should the title be MediaCapture to be consistent with the page naming convention of using the control name. 

If not, the title should be capitalized.

Reply:
The intent was to have the title more directly mirror "CaptureElement", since this demos the alternative which is available in WinAppSDK without CaptureElement.

Capitalizing makes sense.